ros/src/twist_controller/dbw_node.py

```diff
@@ -45,33 +45,54 @@ def __init__(self):
         steer_ratio = rospy.get_param('~steer_ratio', 14.8)
         max_lat_accel = rospy.get_param('~max_lat_accel', 3.)
         max_steer_angle = rospy.get_param('~max_steer_angle', 8.)
+        
 
         self.steer_pub = rospy.Publisher('/vehicle/steering_cmd',
                                          SteeringCmd, queue_size=1)
         self.throttle_pub = rospy.Publisher('/vehicle/throttle_cmd',
                                             ThrottleCmd, queue_size=1)
         self.brake_pub = rospy.Publisher('/vehicle/brake_cmd',
                                          BrakeCmd, queue_size=1)
+                                         
+                                    
 
         # TODO: Create `TwistController` object
-        # self.controller = TwistController(<Arguments you wish to provide>)
+        self.controller = TwistController(vehicle_mass, brake_deadband, decel_limit, accel_limit, wheel_radius, wheel_base, steer_ratio, max_lat_accel, max_steer_angle)
 
         # TODO: Subscribe to all the topics you need to
-
+        self.current_velocity_sub = rospy.Subscriber("/vehicle/current_velocity", TwistStamped, self.current_velocity_callback)
+        self.twist_cmd_sub = rospy.Subscriber("/vehicle/twist_cmd", TwistStamped, self.twist_cmd_callback)
```

Comment:
I think this the wrong subscription (/vehicle/twist_cmd).  From the log at the top of dbw_node.py:

"You will subscribe to `/twist_cmd` message which provides the proposed linear and angular velocities."